config/manager/manager.yaml

```
@@ -33,6 +33,7 @@ spec:
             - --health-probe-bind-address=:8081
             - --metrics-bind-address=0.0.0.0:9090
             - --leader-elect
+            - --max-concurrent-reconciles=1
```

As the default is already 1, having it defined the Kustomize could be a detriment to users who wish to change it.
Removing/changing an entry in an array with patches requires you to either know the exact index or overwrite the entire array.
Hinting at the option being available is to my understanding preferred, correct me if I'm wrong @weisdd 😄 
```suggestion
           #  - --max-concurrent-reconciles=1
```

---

I think we don't necessarily add hints in the same way everywhere, but it's certainly worth documenting somewhere.

We don't have a dedicated page for controller options, so, in my view, it would nice to have the commented out option like @Baarsgaard suggested. In future, we might move such references to docs.

---

Good point. Commented out ;)